contracts/modules/cover/Cover.sol

```diff
@@ -637,6 +637,18 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     }
   }
 
+  function editProductTypes(
+    uint[] calldata productTypeIds,
+    ProductType[] calldata newProductTypes,
+    string[] calldata ipfsMetadata
+  ) external override onlyAdvisoryBoard {
+    
+    for (uint i = 0; i < productTypeIds.length; i++) {
+      _productTypes[productTypeIds[i]] = newProductTypes[i];
+      emit ProductTypeUpserted(productTypeIds[i], ipfsMetadata[i]);
```

Comment:
Can we please please please change the "upsert" part? It's a term that's highly related to databases and it's not the case to use it here. A couple of ideas: `ProductTypeMetadataUpdated`, `ProductTypeMetadataSet`, `ProductTypeSet`, `ProductTypeChanged`

Reply:
Yeah that's fine i think it was chosen because you can use it for both adding and updating but i agree it's confusing